Superlista/Views/AddCollaboratorSheetView.swift

```diff
@@ -115,7 +115,7 @@ struct AddCollaboratorSheetView: View {
                             }
                             
                             self.showShareActionSheet.toggle()
-                            shareSheet(listID: list.id, option: "2", listName: list.title, ownerName: listOwner.name!)
+                            shareSheet(listID: list.id, option: "2", listName: list.title, ownerName: listOwner.name ?? "")
```

Comment:
Sempre que tiver uma "", se for um nome de lista coloca "NovaLista" (ou o localizable equivalente), e quando for um nome de usuário, chamar a função getNickname()